src/openrct2/entity/Balloon.h

```diff
@@ -24,8 +24,9 @@ struct Balloon : EntityBase
     uint8_t colour;
     static void Create(const CoordsXYZ& balloonPos, int32_t colour, bool isPopped);
     void Update();
-    void Pop();
+    void Pop(bool silent = false);
```

Comment:
Does it need a default parameter? I always like not adding one if there are few cases to cover

Reply:
Good point, I removed the default.